src/styles/previewStyles.less

```diff
@@ -109,6 +110,12 @@
 					}
 				}
 
+				#pageRangeControl {
+					// Temporary workaround that fixes styling issues with the PDF Preview Header
```

Comment:
Temporary?  Famous last words.  :-)

Is there something tracking the "real" fix here?  Also, how arbitrary is 300px?  Is that particular number based on anything?